recipes/nginx.rb

```diff
@@ -91,7 +90,7 @@
       mode '0644'
       variables(
         http_port: port,
-        server_name: site_opts['server_name']
+        server_name: site_opts['server_alias'].empty? ? site_name : site_opts['server_alias'].first
```

Comment:
but is it supposed to be in our datastructure, I still think it is derived from the primary alias, the first alias being the server_name, the following being supplementary aliases.